pkg/instrumentation/podmutator.go

```diff
@@ -195,6 +196,7 @@ func NewMutator(logger logr.Logger, client client.Client, recorder record.EventR
 func (pm *instPodMutator) Mutate(ctx context.Context, ns corev1.Namespace, pod corev1.Pod) (corev1.Pod, error) {
 	logger := pm.Logger.WithValues("namespace", pod.Namespace, "name", pod.Name)
 
+	// logger.Info("namespace Name: %s, NamespaceL %s", ns.Name, ns.Namespace)
```

Comment:
should weleave this in?

Reply:
Leftover from my testing. 549cef63daebc58ca74ac9221e78cafcc880c2b9